### DISCUSSION TITLE
Why is .NET reported in Type script project?

### DISCUSSION THREAD

**Discussion by kaihendry** [starship/starship#5585] [Q&A]
` "via .NET " (<1ms)                             -  The relevant version of the .NET Core SDK for the current directory`

I'm not using dotNET!

**Comment by davidkna** [comment]
By default, the module is shown in your prompt when one or more of the following files are present in the current directory:

global.json
project.json
Directory.Build.props
Directory.Build.targets
Packages.props
*.csproj
*.fsproj
*.xproj

**Reply by kaihendry** [reply]
The typescript project uses nx, which uses a project.json

**Reply by davidkna** [reply]
I opened an issue to discuss potentially removing those from the default `dotnet.detect_files` option (#5592).
